features/step_definitions/et1_ccd_steps.rb

```diff
@@ -266,3 +269,28 @@
   expected_names = @respondent[0..number].map { |r| "acas_#{r[:name]}.pdf" }
   expect(ccd_object.find_acas_names(number)).to match_array(expected_names)
 end
+
+And(/^the PDF is converted correctly$/) do
+  office = @respondent[0]["expected_office"]
+  ccd_office_lookup = ::EtFullSystem::Test::CcdOfficeLookUp
+  ccd_object = EtFullSystem::Test::Ccd::Et1CcdSingleClaimant.find_by_reference(@claim_reference, ccd_office_lookup.office_lookup[office][:single][:case_type_id])
+
+  ccd_pdf = ccd_object.find_pdf_attachment
+  reader = PDF::Reader.new(ccd_pdf)
+  pdf_content_first_line = reader.pages[0].text.lines.first.strip
+  pdf_content_second_line = reader.pages[0].text.lines.second.strip
+
+  rtf_text_first_line = nil
+  rtf_text_second_line = nil
+  rtf_file = 'features/support/fixtures/simple_user_with_rtf.rtf'
+  silence_warnings do
+    rtf_content = File.read(rtf_file)
+    parser = RubyRTF::Parser.new
+    rtf_text_first_line = parser.parse(rtf_content).sections.first[:text].strip
+    rtf_text_second_line = parser.parse(rtf_content).sections.second[:text].strip
```

Comment:
You're parsing twice here, which seems like an inefficient approach, surely it would be better to parse once, and strip twice instead?
`  rtf_sections = parser.parse(rtf_content).sections
  rtf_text_first_line = rtf_sections.first[:text].strip
  rtf_text_second_line = rtf_sections.second[:text].strip `